public/data/ayushlohmod.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Ayush Lohmod",
+  "type": "personal",
+  "bio": "I am Ayush Lohmod, currently pursuing Batchelors of Computer Application from SGT UNIVERSITY. I am a tech enthusiast. I am always open to collaborating on projects and innovative/disruptive ideas",
+  "avatar": "https://github.com/ayushlohmod.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/ayushlohmod",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/ayushlohmod_",
```

Comment:
There is a **broken link**.  Please update it to the correct one! 
![image](https://user-images.githubusercontent.com/88786642/177276279-3dd0a577-ece6-4fe0-b756-af9e82495cd9.png)